fitwsarah_frontend/src/components/AdminPanel/Filter.js

```diff
@@ -29,29 +33,62 @@ function Filter({labels, onInputChange, searchTerm, clearFilters}) {
         setStatusButton("");
         clearFilters();
     }
+    //depending on where the filter will be, use this to automatically localize the name
+    const source = (label) => {
```

Comment:
Could you explain why you decided to use the switch loop to iterates through your values? Im curious.

Reply:
Good question! It is to iterate between the different labels from whatever page the filter componenent is on, it will then return the localization key that will then be used in the translation below. That way, we can dynamically add labels in source and return the fr and en version at the same time.